build_tools/cmake/iree_tablegen_doc.cmake

```diff
@@ -56,6 +55,11 @@ function(iree_tablegen_doc)
   set(_INPUTS ${_RULE_TD_FILE})
   set(LLVM_TARGET_DEFINITIONS ${_INPUTS})
 
+  set(_FLAGS
+    # Pass this after https://reviews.llvm.org/D152404
+    # "--strip-prefix=::mlir::iree_compiler::IREE::"
+  )
```

Comment:
Not passing this through to Bazel, considering how empty the Bazel version is: https://github.com/openxla/iree/blob/eb26576d8190600f4c0556e8594469896348a9f5/build_tools/bazel/build_defs.oss.bzl#L151-L160